### PR TITLE
EMP-2651, change condition ctx

### PR DESCRIPTION
**Pull request by Rohitrajak1807**
Fixes: https://platform9.atlassian.net/browse/EMP-2651

### CODE REVIEW

File: emp/emp_user_cftemplate.yaml [hunk 1]
```diff
@@ -335,8 +335,8 @@ Resources:
               StringEquals:
                 aws:RequestTag/emp.pf9.io: owned
               StringLike:
-                aws:RequestTag/emp.pf9.io/baremetalpool: '*'
-                aws:RequestTag/emp.pf9.io/namespace: '*'
+                aws:RequestTag/emp.pf9.io/ns/org-*/bmtpool/*: owned
```
Comment (khallnayak) [inline emp/emp_user_cftemplate.yaml:338]: nit: can we use `bmtpool` instead of `bmpool`

**Review by khallnayak — APPROVED** [review]
Minor nit. rest looks good

File: emp/emp_user_cftemplate.yaml (hunk 1 above)
Comment (Rohitrajak1807) [reply emp/emp_user_cftemplate.yaml:338]: We'll have to change the controllers again and retest

**Review by khallnayak — APPROVED** [review]
LGTMIT

**Pull request merged**